learn/what_is_meilisearch/telemetry.md

```diff
@@ -110,9 +110,8 @@ This list is liable to change with every new version of Meilisearch. It's not be
 | `infos.ignore_missing_dump`                        | `true` if `--ignore-missing-dump` is activated, otherwise `false`                           | true
 | `infos.ignore_dump_if_db_exists`                   | `true` if `--ignore-dump-if-db-exists` is activated, otherwise `false`                      | true
 | `infos.import_snapshot`                            | `true` if `--import-snapshot` is specified, otherwise `false`                               | true
-| `infos.schedule_snapshot`                          | `true` if `--schedule-snapshot`/`MEILI_SCHEDULE_SNAPSHOT` is activated, otherwise `false`   | true
+| `infos.schedule_snapshot`                          | Value of `--schedule_snapshot`/`MEILI_SCHEDULE_SNAPSHOT` if scheduled snapshots are enabled, otherwise `None`   | `86400`
```

Comment:
None of the values in the "Example" column use backticks, but they do in "Description". We should update it.
```suggestion
| `infos.schedule_snapshot`                          | Value of `--schedule_snapshot`/`MEILI_SCHEDULE_SNAPSHOT` if scheduled snapshots are enabled, otherwise `None`   | 86400
```